sdgPlots

lilyclements opened the pull request:
Edited sdgPlots
ucrAdditionalLayer
ucrAxes

Fixed bugs

dannyparsons commented on instat/sdgPlots.vb:
```diff
@@ -13,6 +13,7 @@
 '
 ' You should have received a copy of the GNU General Public License k
 ' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Imports instat
```
What is this needed for?

dannyparsons commented on instat/sdgPlots.vb:
```diff
@@ -331,11 +336,11 @@ Public Class sdgPlots
     End Sub
 
-    Private Sub nudNoOfRowsOrColumns_TextChanged(sender As Object, e As EventArgs) Handles nudNoOfRowsOrColumns.TextChanged
+    Private Sub nudNumberofRows_ValueChanged(sender As Object, e As EventArgs) Handles nudNumberofRows.ValueChanged
```
We can keep this as TextChanged

dannyparsons commented on instat/ucrAxes.vb:
```diff
@@ -49,6 +49,14 @@ Public Class ucrAxes
+        nudFrom.Visible = False
+        nudTo.Visible = False
+        nudInStepsOf.Visible = False
+        nudTickMarkersNoOfDecimalPlaces.Visible = False
```
Visibility shouldn't be a default. It should only be controlled only by methods. Maybe those methods need to run in SetDefaults?

lilyclements commented:
Removed ImportsInstat (not sure why that added in but I've noticed it keeps adding in in a few places)
Made .TextChanged
Removed defaults that were added - realised the issue is coming from elsewhere
